### DISCUSSION TITLE
FlexItem pattern

### DISCUSSION THREAD

**Discussion by ilterugur** [chakra-ui/panda#1030] [Ideas]
### Description

FlexItem pattern would be useful

### Problem Statement/Justification

Many developers are using the traditional row-column system which works on flex display, instead of grid. Grid is usually an overkill solution.

### Proposed Solution or API

Adding FlexItem pattern which will be similar to GridItem in functionality.

**Comment by segunadebayo** [comment]
Hi @ilterugur,

This sounds like something that could be implemented in userland.

I'm curious what the API for a flex item pattern would look like.
